packages/client/modules/summary/components/NewMeetingSummaryRoot.tsx

```diff
@@ -1,37 +1,25 @@
-import graphql from 'babel-plugin-relay/macro'
-import React from 'react'
-import {QueryRenderer} from 'react-relay'
+import React, {Suspense} from 'react'
 import {RouteComponentProps, withRouter} from 'react-router-dom'
-import Atmosphere from '../../../Atmosphere'
-import withAtmosphere from '../../../decorators/withAtmosphere/withAtmosphere'
-import {LoaderSize} from '../../../types/constEnums'
-import renderQuery from '../../../utils/relay/renderQuery'
 import NewMeetingSummary from './NewMeetingSummary'
+import useQueryLoaderNow from '../../../hooks/useQueryLoaderNow'
+import newMeetingSummaryQuery, {
+  NewMeetingSummaryQuery
+} from '../../../__generated__/NewMeetingSummaryQuery.graphql'
+import {LoaderSize} from '../../../types/constEnums'
+import {renderLoader} from '../../../utils/relay/renderLoader'
 
-const query = graphql`
-  query NewMeetingSummaryRootQuery($meetingId: ID!) {
-    viewer {
-      ...NewMeetingSummary_viewer
-    }
-  }
-`
+interface Props extends RouteComponentProps<{urlAction?: 'csv'; meetingId: string}> {}
 
-interface Props extends RouteComponentProps<{urlAction?: 'csv'; meetingId: string}> {
-  atmosphere: Atmosphere
-}
-const NewMeetingSummaryRoot = ({atmosphere, match}: Props) => {
+const NewMeetingSummaryRoot = ({match}: Props) => {
   const {
     params: {urlAction, meetingId = 'demoMeeting'}
   } = match
+  const queryRef = useQueryLoaderNow<NewMeetingSummaryQuery>(newMeetingSummaryQuery, {meetingId})
   return (
-    <QueryRenderer
-      environment={atmosphere}
-      query={query}
-      variables={{meetingId}}
-      render={renderQuery(NewMeetingSummary, {props: {urlAction}, size: LoaderSize.WHOLE_PAGE})}
-      fetchPolicy={'store-or-network' as any}
-    />
+    <Suspense fallback={renderLoader({size: LoaderSize.WHOLE_PAGE})}>
+      {queryRef && <NewMeetingSummary queryRef={queryRef} urlAction={urlAction} />}
+    </Suspense>
   )
 }
 
-export default withAtmosphere(withRouter(NewMeetingSummaryRoot))
+export default withRouter(NewMeetingSummaryRoot)
```

Comment:
+1 `withRouter` is deprecated, you could replace it with `useRouter` if you like, although if this PR is too big, I understand